sql-to-dbsp-compiler/lib/sqllib/src/string.rs

```diff
@@ -318,3 +320,217 @@ pub fn writelog<T: std::fmt::Display>(format: String, argument: T) -> T {
     print!("{}", formatted);
     argument
 }
+
+pub fn md5__(value: String) -> String{
+    let mut  hasher = Md5::new();
+    hasher.input(value.as_bytes());
+    let result = hasher.result();
+    return format!("{:x}", result);
```

Comment:
since you have ownership of `value` you could reuse it as the output String which would probably avoid extra allocations in many cases